collectors/base/record.py

```diff
@@ -105,7 +105,7 @@ def write(self, conf, conn):
         action = 'created'
         if table.find_one(**{self.__primary_key: self[self.__primary_key]}):
             action = 'updated'
-            for key in ['meta_id', 'meta_updated']:
+            for key in ['meta_id', 'meta_created']:
```

Comment:
Both `meta_created` and `meta_updated` [get reset when a record is created](https://github.com/opentrials/collectors/blob/master/collectors/base/record.py#L65). 
If here it turns out the record should be updated then `meta_created` should be removed from the new record in order to avoid updating the `meta_created` of the existing record.